README.md

```diff
@@ -1106,6 +1106,141 @@ Remove a specific collection
 model.remove()
 ```
 
+### Command Model
+```js
+'use strict';
+const Model = require('screwdriver-models');
+const factory = Model.CommandFactory.getInstance({
+    datastore
+});
+const config = {
+    namespace: 'testCommandNS',
+    command: 'testCommand',
+    version: '1.3',
+    description: 'This is a test command',
+    maintainer: 'foo@bar.com',
+    format: 'habitat',
+    habitat: {
+        mode: 'remote',
+        package: 'core/git/2.14.1',
+        binary: 'git'
+    },
+    labels: ['beta', 'stable']
+}
+
+factory.create(config)
+    .then(model => {    // do something
+    });
+```
+
+#### Update
+Update a specific command
```

Comment:
This PR states that it will only take care of CREATE and GET. We shouldn't have this here yet.

Reply:
I removed it. Thanks.